docs/source/overview/cli.md

```diff
@@ -0,0 +1,499 @@
+- `test_predictions.npz`: The model predictions for each sample in the test set. Useful if you want to make your own visualizations/figures.
+- `test_targets.npz`: The actual classes for each sample in the test set. Useful if you want to make your own visualizations/figures.
+- `precision_recall_curves.svg`: An AUPRC figure that we generate for you. Each curve corresponds to one of the classes the model predicts.
+- `roc_curves.svg`: An ROC AUC figure that we generate for you. Each curve corresponds to one of the classes the model predicts.
+- `selene_sdk.evaluate_model.log`: Note that if `evaluate` is run through `train_model` (that is, no `evaluate_model` configuration was specified, but you used `ops: [train, evaluate]`) you will only see `selene_sdk.train_model.log`. `selene_sdk.evaluate_model.log` is created when `evaluate_model` is used and will output some logging information related to the `selene_sdk.EvaluateModel` class (some debug statements and performance metrics). 
+
+## Analyze sequences
+The `analyze` operation allows you to apply a trained model to new sequences of interest. Currently, we support 3 "sub-operations" for `analyze`:
+
+1) Prediction on sequences: Output the model predictions for a list of sequences.
+2) Variant effect prediction: Output the model predictions for sequences centered on specific variants (will output reference and alternate predictions as separate files). 
+3) _In silico_ mutagenesis: _In silico_ mutagenesis (ISM) involves computationally "mutating" every position in the sequence to every other possible base (DNA and RNA) or amino acid (protein sequences) and examining the consequences of these "mutations". For ISM, Selene outputs the model predictions for the reference (original) sequence along with each of the mutated sequences. 
+ 
+For variant effect prediction and _in silico_ mutagenesis, a number of scores can be computed using the predictions from the reference and alternate alleles. You may select 1 or more of the following as outputs:
+- `predictions` (output the predictions for each variant, as described above)
+- `diffs` (difference scores): The difference between alt and ref predictions.
+- `abs_diffs` (absolute difference scores): The absolute difference between alt and ref predictions.
+- `logits` (log-fold change scores): The difference between `logit(alt)` and `logit(ref)` predictions.
+You'll find examples of how this is specified in the [variant effect prediction](#Variant-effect-prediction) and [_in silico_ mutagenesis](#In-silico-mutagenesis) sections.
+
+In all `analyze`-related operations, we ask that you specify 2 configuration keys. One will always be the `analyze_sequences` key, which we will explain generally here. The other one is dependent on which of the 3 sub-operations you use and will be explained in the appropriate subsection below. 
```

Comment:
I would remove the 'explain generally here' and just proceed without noting when you will talk about them. You are talking about those in a consecutive manner, so I do not think it would cause a confusion. 
```suggestion
In all `analyze`-related operations, we ask that you specify 2 configuration keys. One will always be the `analyze_sequences` key and the other one is dependent on which of the 3 sub-operations you use - `prediction`, `variant_effect_prediction` or `in_silico_mutagenesis`. 

### Analyze sequences

```